common/lib/dataset.py

```diff
@@ -1510,6 +1502,20 @@ def is_from_collector(self):
 		"""
 		return self.type.endswith("-search") or self.type.endswith("-import")
 
+	def file_exists(self):
```

Comment:
Since this is just used in one place why not call `dataset.get_results_path().exists()` directly in the template?